### PR TITLE
Add support of TOFS

### PR DESCRIPTION
**Pull request by n-rodriguez**

### CODE REVIEW

**Review by aboe76 — APPROVED** [review]
Nice Work @n-rodriguez

**Comment by myii** [comment]
Diff before and after (map and `file.managed` x1):

```diff
@@ -1,13 +1,25 @@
-https://travis-ci.org/myii/exim-formula/jobs/593914331#L1263-L1285
+https://travis-ci.org/myii/exim-formula/jobs/593914361#L1263-L1298
        [DEBUG   ] Map   : .../exim-formula/.kitchen/yaml_dump/Debian-10-2019.2.0-n/a-e683178-py3.yaml
        aliases:
          root: root@bar.baz
        arch: amd64
        config:
+         cfilemode: '644'
          configtype: satellite
          hide_mailname: 'false'
+         local_interfaces: ''
+         localdelivery: mail_spool
+         minimaldns: 'false'
+         other_hostnames: ''
+         readhost: ''
+         relay_domains: ''
+         relay_nets: ''
+         smarthost: ''
+         ue4c_keepcomments: 'true'
+         use_split_config: 'true'
        config_dir: /etc/exim4
        config_file: update-exim4.conf.conf
+       files: {}
        mailname: foo.bar.baz
        mailname_file: /etc/mailname
        pkgs:
@@ -21,23 +33,15 @@
        - retry
        - rewrite
        - auth
+       virtual_aliases: {}
        virtual_aliases_dir: /etc/exim4/virtual
 
 
-https://travis-ci.org/myii/exim-formula/jobs/593914331#L1425-L1440
+https://travis-ci.org/myii/exim-formula/jobs/593914361#L1450-L1456
-       /etc/exim4/update-exim4.conf.conf:
+       exim/config/install:
          file.managed:
-           - contents: |
-        dc_eximconfig_configtype='satellite'
-        dc_other_hostnames=''
-        dc_local_interfaces=''
-        dc_relay_domains=''
-        dc_relay_nets=''
-        dc_use_split_config='true'
-        dc_hide_mailname='false'
-        ue4c_keepcomments='true'
-        dc_localdelivery='mail_spool'
-        dc_minimaldns='false'
-        dc_readhost=''
-        dc_smarthost=''
-        CFILEMODE='644'
+           - name: /etc/exim4/update-exim4.conf.conf
+           - source: 
+             - salt://exim/files/7bdbc125c90f/update-exim4.conf.conf
+             - salt://exim/files/Debian/update-exim4.conf.conf
+             - salt://exim/files/default/update-exim4.conf.conf
```

* All looks good.

File: pillar.example [hunk 1]
```diff
@@ -45,3 +45,38 @@ exim:
...
+    #   exim/config/install:
+    #     - 'example_alt.tmpl'
+    #     - 'example_alt.tmpl.jinja'
+
+    # For testing purposes
+    source_files:
+      exim/config/install:
+        - 'example.tmpl.jinja'
+      exim/subcomponent/config/install:
+        - 'subcomponent-example.tmpl.jinja'
```
Comment (myii) [inline pillar.example]: Just needs some tidying up here.

**Review by myii — CHANGES_REQUESTED** [review]
Minor change, I'll probably make it and get this merged.

---

* Checked using https://github.com/myii/ssf-formula/pull/57

File: pillar.example (hunk 1 above)
Comment (myii) [reply pillar.example]: @n-rodriguez Done (amended the last commit and force pushed).

**Review by myii — APPROVED** [review]

**Comment by myii** [comment]
Lovely work again, @n-rodriguez!  Thanks for all of your efforts across all formulas recently.  I'll merge this as soon as the tests pass.

**Comment by myii** [comment]
@n-rodriguez Done.

**Pull request merged**